contracts/DepositAccount.sol

```diff
@@ -22,9 +24,13 @@ contract DepositAccount is IAccount {
         // Account should deposit missing funds to EntryPoint to pay gas for the user operation.
         //
         // HINT: There are two possible ways to achieve this goal:
+        //
         // (1) Send ETH directly to EntryPoint
+        // * Sending ETH in Solidity: https://solidity-by-example.org/sending-ether/
+        // * Convert interface to address payable, for example, `payable(address(entryPoint))`
+        //
         // (2) Call `depositTo` on EntryPoint
-        // Check out the EntryPoint contract: https://github.com/eth-infinitism/account-abstraction/blob/develop/contracts/core/EntryPoint.sol
+        // * Check out the EntryPoint interface: https://github.com/eth-infinitism/account-abstraction/blob/dae9733bf78bcb7576f572d67497c2d241ae4da1/contracts/interfaces/IStakeManager.sol#L76-L80
```

Comment:
StakeManager Interface?
```suggestion
        // * Check out the StakeManager interface: https://github.com/eth-infinitism/account-abstraction/blob/dae9733bf78bcb7576f572d67497c2d241ae4da1/contracts/interfaces/IStakeManager.sol#L76-L80
```

Reply:
Fixed in [1bece53](https://github.com/consenlabs/ethtaipei2023-aa-workshop/pull/10/commits/1bece53a397b9e4e93385a1ae9b5f3f56e29d1dc)